docs/Project.toml

```diff
@@ -10,9 +10,9 @@ Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 SolverBenchmark = "581a75fa-a23a-52d0-a590-d6201de2218a"
 
 [compat]
-ADNLPModels = "0.6"
-Documenter = "0.27"
-JSOSolvers = "0.9"
-NLPModels = "0.19"
+ADNLPModels = "0.7"
+Documenter = "1.0"
```

Comment:
Why 1.0 and not 1?

Reply:
I like to put major and minor version numbers for aesthetic reasons :)

Reply:
Ok, the advantage of putting just 1 is that we no longer have to track 1.1, 1.2, etc

Reply:
I don't think so Tangi: https://pkgdocs.julialang.org/v1/compatibility/#Caret-specifiers

Reply:
Oh, you are right, thanks for that!